app/plugin.go

```diff
@@ -662,16 +674,54 @@ func (a *App) installPrepackagedPlugin(pluginPaths *pluginSignaturePaths) (*plug
 	}
 	defer os.RemoveAll(tmpDir)
 
-	plug, pluginDir, err := getPrepackagedPlugin(pluginPaths, fileReader, tmpDir)
+	plugin, pluginDir, err := getPrepackagedPlugin(pluginPaths, fileReader, tmpDir)
 	if err != nil {
 		return nil, errors.Wrapf(err, "Failed to get prepackaged plugin %s", pluginPaths.path)
 	}
 
-	if _, err := a.installExtractedPlugin(plug.Manifest, pluginDir, true); err != nil {
-		return nil, errors.Wrapf(err, "Failed to install extracted prepackaged plugin %s", pluginPaths.path)
+	if !*a.Config().PluginSettings.AutomaticPrepackagedPlugins {
```

Comment:
If we don't need to return the `plugin` below, can we move this check much earlier instead?